.env.example

```diff
@@ -0,0 +1 @@
+GOOGLE_MAPS_API_KEY=
```

Comment:
Rename this `.env.example` file to just `.env`. A `.env` file is very common for storing secrets locally, and is usually omitted from version control (`.env.example` only shows you what keys you need, not the values).